alerting/handler.go

```diff
@@ -0,0 +1,9 @@
+package alerting
+
+import (
+	"github.com/datajet-io/peekaboo/services"
+)
+
+type AlertHandler interface {
```

Comment:
If it triggers an alert, its not a handler its more like a trigger? (now this entity conflicts a bit with entity in service code below).